pkg/signatures/engine/engine.go

```diff
@@ -344,3 +367,36 @@ func (engine *Engine) GetSelectedEvents() []detect.SignatureEventSelector {
 	}
 	return res
 }
+
+func (engine *Engine) RegisterDataSource(dataSource detect.DataSource) error {
```

Comment:
Please improve godocs for exported methods (All of them if possible).

Reply:
TODO: check if you want to address this after freeze (freezing tomorrow). This will be merged as is by https://github.com/aquasecurity/tracee/pull/3142